manifests/n/NHNCorporation/Dooray!Messenger/.validation

```diff
@@ -0,0 +1 @@
+{"ValidationVersion":"1.0.0","Waivers":[{"WaiverId":"73eb1894-14df-4787-aa90-51c75accd048","TestPlan":"Validation-Domain","PackagePath":"manifests/n/NHNCorporation/Dooray!Messenger/1.1.5.1"}]}
```

Comment:
The installer file for 1.1.5.1 appears to be getting deleted in the linked PR. If the source material for the waiver is purged, this waiver will become irrelevant.